example/index.android.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+import { TwitterLoginButton } from 'react-native-twitter-sdk'
+
+export default class example extends Component {
+
+  state = {
+    twitter_access_token: '',
+    twitter_token_secret: '',
+    twitter_userId: '',
+    error_message: '',
+  }
+
+  onTwitterLoginFinished = (error, result) => {
+    console.log('onTwitterLoginFinished in REACT')
+    if (error) {
+      alert("login has error: " + error);
+         this.setState({
+           error_message: result.error,
+         })
+    } else if (result.isCancelled) {
+      alert("login is cancelled.");
+    } else {
+    console.log(result)
+          this.setState({
+            twitter_access_token: result.authToken.token,
+            twitter_token_secret: result.authToken.secret,
+            twitter_userId: result.userId,
+          })
+    }
+  }
+
+  render() {
+
+  const { twitter_access_token, twitter_token_secret, twitter_userId, error_message } = this.state
+
+    return (
+      <View style={styles.container}>
+      <TwitterLoginButton
+           accessible
+           accessibilityLabel={'loginButton'}
+           onLoginFinished={this.onTwitterLoginFinished}
+           onLogoutFinished={() => alert("logout.")}/>
+        <Text
+           accessibilityLabel="twitter_response"
+           style={styles.instructions}>
+          { twitter_access_token != '' ? 'twitter_access_token: ' + twitter_access_token : ''} {'\n'}
```

Comment:
First of all:
1. `twitter_access_token` => `twitterAccessToken`
2. `!=` => `!==`
3. 
```js
'twitter_access_token: ' + twitter_access_token
``` 
=> 
```js 
`twitter_access_token: ${twitter_access_token}`
```